docker-compose.yml

```diff
@@ -1,27 +1,30 @@
 version: "3"
 services:
-
   api:
+    extends:
+      file: docker-compose.common.yml
+      service: common
     ports:
       - "8800:8800"
-    build: .
-    command: ["sh", "-c", "yarn mongo-indexes && yarn api"]
-    restart: on-failure
+    command: ["sh", "-c", "yarn api"]
```

Comment:
was `yarn mong-indexes` taken out from the startup command chain on purpose?

Reply:
Good point, I will check it

Reply:
Pls let me know how did this go & I'm going to approve it from my part.

Reply:
I put it back... I dont know why I removed it in the first place .... by mistake